app/components/autocomplete/channel_mention/index.ts

```diff
@@ -79,13 +81,30 @@ const withTeamId = withObservables(['teamId', 'channelId'], ({teamId, channelId,
     };
 });
 
+function getMatchPattern(isSearch: boolean, database: Database) {
```

Comment:
This returns an observable, so it should be called `observeX` instead of `getX`.